addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5625,27 +5625,67 @@ msgstr ""
 
 #empty strings from id 10802 to 10819
 
+#. Title of controller configuration window
```

Comment:
Please also provide a reference to the source file(s) using the new strings. `#: /xbmc/...`